### PR TITLE
added Load_save service for database manager

### PR DESCRIPTION
**Pull request by shipra25jain**
Input is the name of file (with path). To save the database, set the toSave as true and to load it from some database on disk, set toSave as False.

### CODE REVIEW

File: database_manager/src/run_server.cpp [hunk 1]
```diff
@@ -1944,6 +1945,60 @@ bool plot_facts_db(toaster_msgs::PlotFactsDB::Request &req, toaster_msgs::PlotFa
 	} 
 }
 
+//////////////////////////////////////////
```
Comment (Greg8978) [inline database_manager/src/run_server.cpp]: please remove dead code (commented one) from your commit.

Comment (shipra25jain) [inline database_manager/src/run_server.cpp]: Removed.

File: toaster_msgs/srv/LoadSaveDB.srv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+string file_name
```
Comment (Greg8978) [inline toaster_msgs/srv/LoadSaveDB.srv]: please use fileName instead of file_name to improve consitency

Comment (shipra25jain) [inline toaster_msgs/srv/LoadSaveDB.srv]: Done.

**Comment by Greg8978** [comment]
Ok, thanks for the modifications, can you squash the commits?
Thanks :)

**Comment by shipra25jain** [comment]
squashed.

**Comment by Greg8978** [comment]
Well, it's not up-to-date anymore...
Maybe you forgot to rebase?

**Pull request closed without merging**